hw/ip/otbn/dv/smoke/run_smoke.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Runs the OTBN smoke test (builds software, build simulation, runs simulation
+# and checks expected output)
+
+set -o pipefail
+
+if [ ! -f './util/build_consts.sh' ]; then
+  echo "Must be run from OpenTitan repository root"
```

Comment:
I think that error messages should be echoed to standard error: `echo >&2 "aargh"` (here and below). You can avoid some of the tedious repetition by defining a "fail" function. For example (untested):

```bash
fail() {
    echo >&2 "$@"
    exit 1
}

...

cmd that could go wrong || fail "oh dear!"
if [ x"$foo" != xfoo ]; then
  fail "oh dear!"
fi

```